components/js/signal-projects.js

```diff
@@ -413,7 +413,13 @@ function createMarkers(data, style) {
         var lat = data[i].location_latitude;
         
         var lon = data[i].location_longitude;
-         
+        
+        if (data[i].construction_note_date) {
+            console.log(data[i], formats.formatDate( new Date(data[i].construction_note_date) ));
+        } else {
+            console.log(data[i], "no date");
+        };
```

Comment:
Can we remove all of the `console.log`s before you merge?

Reply:
understand it helps with debugging and testing but we shouldn't leave it in there because it'll make debugging more tricky in the future.

Reply:
Totally understand - I generally remove my debugging/testing code but I missed that one. Sorry about that, will make sure to get rid of it before I merge!